config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -4296,6 +4296,13 @@
     <lineContent>private Deque&lt;Counter&gt; counters;</lineContent>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java</fileName>
+    <specifier>initialization.field.uninitialized</specifier>
+    <message>the default constructor does not initialize field currentRangeValue</message>
```

Comment:
@romani @nrmancuso @Vyom-Yadav This check conflicts with pitest when it is the default value is assigned. I am sure we have a check to also avoid default assignments as they are already defaulted to when we don't assign them.
Pitest reasoning at https://github.com/checkstyle/checkstyle/pull/12639#discussion_r1070235437